chainlink-nodes.md

```diff
@@ -6,6 +6,11 @@ title: "Welcome to Chainlink Node Operation!"
 
 <div>
   <div class="markdown-body">
+    <div class="cl-section-header">
+      <p>
+        Chainlink provides reliable, tamper-proof inputs and outputs for complex smart contracts on any blockchain. The following pages contain comprehensive documentation of the Chainlink ecosystem. If you are new to Chainlink, you might want to check out the <a href="/docs/architecture-overview">Architecture Overview</a> first.
```

Comment:
We need to explain the problem first, then how Chainlink solves it.

Reply:
Are you sure this has to be on the welcome page of the docs? I can refer to the [oracle problem post](https://blog.chain.link/what-is-the-blockchain-oracle-problem/) as a link if they don't know anything about it. I just haven't seen any dev portal that starts with the problem statement. Maybe we should add the oracle problem into the docs website to avoid people jumping around to the blog?